content/en/security_platform/cloud_workload_security/guide/tuning-rules.md

```diff
@@ -0,0 +1,188 @@
+---
+title: Fine-tuning CWS security signals
+kind: guide
+---
+
+## Overview	
+
+Datadog Cloud Workload Security monitors suspicious activity occurring at the workload level. However in some cases, benign activities are flagged as malicious because of particular settings in the user's environment. When a benign expected activity is triggering a signal, you can suppress the trigger on the activity to limit noise. 
```

Comment:
```suggestion
Datadog Cloud Workload Security monitors suspicious activity occurring at the workload level. However, in some cases, benign activities are flagged as malicious because of particular settings in the user's environment. When a benign expected activity is triggering a signal, you can suppress the trigger on the activity to limit noise. 
```